src/Html2OpenXml/HtmlConverter.cs

```diff
@@ -120,37 +120,80 @@ public IList<OpenXmlCompositeElement> Parse(String html)
 			return paragraphs;
 		}
 
-        /// <summary>
+		/// <summary>
 		/// Start the parse processing and append the converted paragraphs into the Body of the document.
 		/// </summary>
-        public void ParseHtml(String html)
-        {
-            // This method exists because we may ensure the SectionProperties remains the last element of the body.
-            // It's mandatory when dealing with page orientation
-
-            var paragraphs = Parse(html);
+		public void ParseHtml(String html) {
+			// This method exists because we may ensure the SectionProperties remains the last element of the body.
+			// It's mandatory when dealing with page orientation
+
+			var paragraphs = Parse(html);
+
+			Body body = mainPart.Document.Body;
+			SectionProperties bodySectionProperties = body.GetLastChild<SectionProperties>();
+			bodySectionProperties?.Remove();
+			var firstparaSectionProps = body.Descendants<SectionProperties>().FirstOrDefault();
+			var lastSectionIsPaginated = false;
+			PageNumberType pageNumType = null;
+			FooterReference footerRef = null;
+			//if we find a pageNumType and the last section is paginated the most likely we need to pass it on to the first added para.
+			if (!(bodySectionProperties is null)) {
+				pageNumType = bodySectionProperties.GetFirstChild<PageNumberType>();
+				pageNumType?.Remove();
+				footerRef = bodySectionProperties.GetFirstChild<FooterReference>();

```

Comment:
should the footerRef be removed from bodySectionProperties as well (if we move somewhere else), similar to pageNumType?
**WARN: there is a check for it below.**

Reply:
Indeed, the check should be done prior to removing tho.